cli/dcoscli/marathon/main.py

```diff
@@ -682,7 +683,7 @@ def _clean_up_resource_definition(properties):
             # because a user should not specify them. When we do a `get` to see
             # the current schema, we may see these fields, but leaving them
             # will cause us to unnecessarily fail the schema validation.
-            elif not k.startswith("task") and k != "lastTaskFailure":
+            elif not k.startswith("task") and k not in ignore_properties:
```

Comment:
Would like to see the various `task*` properties explicitly listed in `ignore_properties`.